docs/setup/prerequisites.md

```diff
@@ -143,7 +143,7 @@ Core:
   Download and install it with:
 
   ```
-  export GOFUMPT_VERSION=abc0db2c416aca0f60ea33c23c76665f6e7ba0b6
+  export GOFUMPT_VERSION=v0.1.1
   GO111MODULE=on ${OASIS_GO:-go} get mvdan.cc/gofumpt@${GOFUMPT_VERSION}
   GO111MODULE=on ${OASIS_GO:-go} get mvdan.cc/gofumpt/gofumports@${GOFUMPT_VERSION}
```

Comment:
Apparently, `gofumports` is deprecated https://github.com/mvdan/gofumpt/commit/179728cc07226dd410dc11945b2d9917194f9c5a and removed from gofumpt code base https://github.com/mvdan/gofumpt/commit/38fc491470bae6f44e2d38b06277dd95cf1bdf97.

Should we simply remove it here and in the `make fmt` target and rely on people using `gopls` or similar to do proper imports formatting?

Reply:
For golangci-lint, we could add the following [`goimports` configuration](https://golangci-lint.run/usage/linters/#goimports) to check it:

```yaml
linters-settings:
  goimports:
    # Put local imports after 3rd-party packages.
    local-prefixes: github.com/oasisprotocol/oasis-core
```

Reply:
Hm ok sure, although `v0.1.1` version of `gofumports` exists and is published. 

Note that there are minor differences in how `gofumports` and `goimports` handle some things:
- https://github.com/golang/go/issues/31646#issue-436515827
- and https://github.com/golang/go/issues/31646#issuecomment-488164935

`gofumports` handled both of these "correctly"